spec/jobs/irs_attempts_events_batch_job_spec.rb

```diff
@@ -169,4 +303,15 @@
       end
     end
   end
+
+  describe '#reasonable_timespan' do
+    it 'returns true' do
```

Comment:
Sorry, me again with silly nits. 😬 

I'm pretty sure we wrote this together when we were pairing so I feel bad only thinking of this now, but I would make it clear _why_ it returns true or false.

Either:
`it returns true for yesterday` and `it returns false for a week ago`, or with something like a `context 'when the date is 1 day in the past' do` block.